src/fraqture/photo_countdown.clj

```diff
@@ -14,23 +15,32 @@
 
 (def vertical-line-offset 176)
 
+(defn now [] (System/currentTimeMillis))
+
 (defn draw-state [state]
   (apply q/background (:background-color state))
   (q/fill 255 255 255)
   (q/text-size 512)
+  (q/delay-frame 1000)
   (q/text-align :center :center)
   (let [options (:options state)
         serial  (:serial options)
         center-width (/ (q/width) 2)
         center-height (/ (q/height) 2)]
     (if (> (:time-left state) 0)
-      (q/text (str (:time-left state)) center-width center-height)
+      (q/text (str (:time-left state)) center-width center-height))
+    (when (< (:time-left state) 1)
+      (q/text "" center-width center-height)
       (led/paint-window serial 0 0 led/row-count led/col-count [255 255 255]))
-    (q/delay-frame 1000)))
+    (led/refresh serial)
+
+    (if (= (:time-left state) -1)
+      (shell/sh "imagesnap" "-w" "1 " (str "images/" (now) ".jpg")))
```

Comment:
I had trouble identifying how the `imagesnap` dependency is managed -- is the intention that I install it manually, or am I missing something?